faqs/atm7_http.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to enable HTTP on ATM7
+search: atm7
+---
+
+**Singleplayer**
+1. Press the windows key and R at the same time
+2. Type in ``%appdata%\.minecraft``
+3. Go into ``saves`` then the name of your save/world
+4. Then go into serverconfig and search for ``computercraft-server.toml``
+5. Open the file in notepad or any other text editor and search for ``[http]``
+6. There you should find ``enabled``, that should be set to false
+7. Set the option to true and save the file
+8. You're done
```

Comment:
```markdown
1. In CurseForge, go to "My Modpacks" and right-click the ATM-7 modpack.
2. Select "Open Folder" in the dropdown menu.
3. Open the folder titled `saves`.
4. Find your world folder, and open it.
5. Find a folder titled `serverconfig`, and open it.
6. Search for a file named `computercraft-server.toml`, and open it in a text editor.
7. Search the file for a section starting with `[http]`.
8. Find `enabled` within this section, and set it to `true`.
9. Save the file and close the editor, then save and quit to title (or `stop` the server), then relaunch the world.
```

Reply:
Unfortunately github is not allowing me to select multiple lines in the review for some reason, so I cannot use a github suggestion to make this look better. Pain.

Reply:
I would avoid assuming that they are using curseforge.
``1. Open your modpack's `.minecraft` folder. If you are using a launcher like CurseForge then you may have an option to "Open Folder" if you right-click on the modpack profile.``

Reply:
True I suppose. The original idea was for ATM-7 only, but having this for more than just ATM-7 would probably be beneficial.